packages/doenetml-worker-rust/lib-js-wasm-binding/src/math.ts

```diff
@@ -0,0 +1,521 @@
+// @ts-ignore
+import me from "math-expressions";
+
+import { vectorOperators } from "@doenet/utils";
+
+export var appliedFunctionSymbolsDefault = [
+    "abs",
+    "exp",
+    "log",
+    "ln",
+    "log10",
+    "sign",
+    "sqrt",
+    "erf",
+    "cos",
+    "cosh",
+    "acos",
+    "acosh",
+    "arccos",
+    "arccosh",
+    "cot",
+    "coth",
+    "acot",
+    "acoth",
+    "arccot",
+    "arccoth",
+    "csc",
+    "csch",
+    "acsc",
+    "acsch",
+    "arccsc",
+    "arccsch",
+    "sec",
+    "sech",
+    "asec",
+    "asech",
+    "arcsec",
+    "arcsech",
+    "sin",
+    "sinh",
+    "asin",
+    "asinh",
+    "arcsin",
+    "arcsinh",
+    "tan",
+    "tanh",
+    "atan",
+    "atan2",
+    "atanh",
+    "arctan",
+    "arctanh",
+    "arg",
+    "min",
+    "max",
+    "mean",
+    "median",
+    "floor",
+    "ceil",
+    "round",
+    "sum",
+    "prod",
+    "variance",
+    "std",
+    "count",
+    "mod",
+    "re",
+    "im",
+    "det",
+    "trace",
+    "nPr",
+    "nCr",
+];
+
+export var appliedFunctionSymbolsDefaultLatex = [
+    "abs",
+    "exp",
+    "log",
+    "ln",
+    "log10",
+    "sign",
+    "sqrt",
+    "erf",
+    "cos",
+    "cosh",
+    "acos",
+    "acosh",
+    "arccos",
+    "arccosh",
+    "cot",
+    "coth",
+    "acot",
+    "acoth",
+    "arccot",
+    "arccoth",
+    "csc",
+    "csch",
+    "acsc",
+    "acsch",
+    "arccsc",
+    "arccsch",
+    "sec",
+    "sech",
+    "asec",
+    "asech",
+    "arcsec",
+    "arcsech",
+    "sin",
+    "sinh",
+    "asin",
+    "asinh",
+    "arcsin",
+    "arcsinh",
+    "tan",
+    "tanh",
+    "atan",
+    "atan2",
+    "atanh",
+    "arctan",
+    "arctanh",
+    "arg",
+    "min",
+    "max",
+    "mean",
+    "median",
+    "floor",
+    "ceil",
+    "round",
+    "sum",
+    "prod",
+    "variance",
+    "std",
+    "count",
+    "mod",
+    "Re",
+    "Im",
+    "det",
+    "trace",
+    "nPr",
+    "nCr",
+];
+
+let allowedLatexSymbols = [
+    "alpha",
+    "beta",
+    "gamma",
+    "Gamma",
+    "delta",
+    "Delta",
+    "epsilon",
+    "zeta",
+    "eta",
+    "theta",
+    "Theta",
+    "iota",
+    "kappa",
+    "lambda",
+    "Lambda",
+    "mu",
+    "nu",
+    "xi",
+    "Xi",
+    "pi",
+    "Pi",
+    "rho",
+    "sigma",
+    "Sigma",
+    "tau",
+    "Tau",
+    "upsilon",
+    "Upsilon",
+    "phi",
+    "Phi",
+    "chi",
+    "psi",
+    "Psi",
+    "omega",
+    "Omega",
+    "partial",
+    "varnothing",
+    "emptyset",
+    "angle",
+    "circ",
+    "$",
+    "%",
+];
+
+export var textToAst = new me.converters.textToAstObj({
+    appliedFunctionSymbols: appliedFunctionSymbolsDefault,
+});
+
+export function getTextToMathConverter({
```

Comment:
The common name of the types of functions are `factories`. Perhaps `textToMathFactory`?